tasks/apm-host-injection.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Only the Agent 7 repositories contain APM packages. We should check that `datadog_agent_major_version | int > 6` here and fail if it's not.

Reply:
Another thing to check that I just realized is that we want to make sure we're running on a supported platform, which right now is Debian/Ubuntu and RH-based distros (so not SUSE or Windows).

All these checks should be probably done early in the role execution to ensure we fail fast instead of failing the role at the end and leaving a half-configured host.

Reply:
Updated PR!